src/rmodels.c

```diff
@@ -4671,7 +4671,7 @@ static Image LoadImageFromCgltfImage(cgltf_image *cgltfImage, const char *texPat
 {
     Image image = { 0 };
 
-    if (cgltfImage->uri != NULL)     // Check if image data is provided as a uri (base64 or path)
+    if (cgltfImage->uri != NULL)     // Check if image data is provided as an uri (base64 or path)
```

Comment:
I guess the problem here is how we thing "uri" is pronounced.   The "u" is not always a vowel.  I am also not dying here.  However the W3C specifications for URIs (including URLs) use "a url" and "a uri."  I presume because "a universal resource locator" is correct. (I once worked on a Univac computer. :).

Reply:
Ok, let's use a then.